### ISSUE TITLE
pre_safe_import_module AttributeError

### ISSUE THREAD

**Issue by reubengann** [pyinstaller/pyinstaller#3432]
Using the latest developer version (v3.3.1-123-g6d91c16) I am getting a build time exception:

```
File "c:\anaconda3\envs\avirtual\lib\site-packages\PyInstaller\hooks\pre_safe_import_module\hook-setuptools.extern.six.moves.py", line 34, in pre_safe_import_module
    for real_module_name, six_module_name in real_to_six_module_name.items():
AttributeError: 'str' object has no attribute 'items'
```

No such error is generated when running on the latest release 3.3.1. However, I was having an issue with 3.3.1 not including Qt files properly, which was apparently fixed in a more recent commit.

System is
Windows 7 64-bit
Anaconda/Python 3.6.4 (in a virtual environment)
Relevant package versions (I think): PyInstaller (3.4.dev0+6d91c1655), altgraph (0.15), future (0.16.0), macholib (1.9), pypiwin32 (223), pywin32 (223), pywin32-ctypes (0.1.2), six (1.11.0)

Unfortunately I cannot share the code. Can someone help me troubleshoot?

**Comment by tallforasmurf** [comment]
Re PyQt, try the branch at #3233. I note the code with the exception relates specifically to six. Is it possible to do without six?

**Comment by htgoebel** [comment]
setuptools are moving quickly. Thus, two options:
- quick one: pleas update to at latest setuptools version
- sustainable one: run pyinstaller under pdb control and track this down.

> Unfortunately I cannot share the code.

Commercial support is available, see http://www.pyinstaller.org/support.html

**Comment by htgoebel** [comment]
Duplicate of #3806. Should be fixed now in 0ffa574bf87e2e37b0df2a4e2d60cb616f9ff2f2

**Comment by mhbdev** [comment]
Run command as administrator then the pyinstaller command!
That worked for me :/
